sites/main-site/src/content/blog/2024/maintainers-minutes-2024-10-25.mdx

```diff
@@ -0,0 +1,140 @@
+
+:::info
+This system only applies to modules nf-test, and should not be used at pipeline level!
+:::
+
+## Stalled PRs due to waiting for review
+
+It was raised that regularly get almost-complete module PRs, but sometimes they stall because a reviewer leaves comments but does not come back to re-review.
+Even if the module author receives a second approving review, but the author often (because we are nice here at nf-core) doesn't feel comfortable merging without getting the approval from the original reviewer.
+
+After a short discussion we felt that we can add a new reviewing guideline that if all (reasonable) comments have been addressed, and an approval has been given by another reviewer, a PR should be merged after 3 months if the original reviewer did not re-review.
+
+If the original reviewer was temporarily unable to review during this period (which can happen!), the module can still be corrected in a follow up review.
+
+## Replacing monolithic `conf/modules.conf` into per module-configs
+
+A contentious point was brought up by the developers of [nf-core/rnaseq](https://nf-co.re/rnaseq) and [nf-core/methylseq](https://nf-co.re/methylseq).
+
+Both pipelines recently have experimented with modifying the ways in which modules and processes can get customised by config files.
+Currently, the nf-core template has a single file - `conf/modules.conf` - where all module-level configurations occur (specifying additional arguments, file publication specifications etc.).
+
+The two pipelines have instead tried to split this single monolithic file into multiple, with one config file per module e.g. [in this nf-core/methylseq PR](https://github.com/nf-core/methylseq/commit/7b276c97589b65153ae23b34fcf4f6da86bb8d70).
+
+One of the main aims of this is to make CI testing more efficient, by modifications to a config only triggering related tests, rather than the whole suite of tests (as all modules are reliant on the one config file).
+Another reason is to make modules and configurations even more modular and thus portable.
+
+This proposal initially brought up a lot of consternation with most of the other maintainers present.
+The primary reason for this is the fear of findability by users, such as if they want to see which parameters of the tool is customised within the module.
+Currently we can point them to a single file within a single directory, and they can search that.
+By splitting into potentially hundreds of files, there was a fear it'll make it much harder for users to discover.
+Furthermore, the system is not currently 'natively' supported by Nextflow (where modules can automatically detect and load `nextflow.config` files next to the `main.nf`), meaning a lot of manual work to manually `include` the config for each module.
```

Comment:
```suggestion
Additionally, this approach isn’t currently supported by Nextflow’s native capabilities, which would allow modules to automatically detect and load nextflow.config files alongside main.nf. As a result, a substantial amount of manual work would be needed to manually include each module’s config.
```